proto/config/vm.proto

```diff
@@ -94,4 +94,7 @@ message VmConfig {
   bool enforce_network_interface_order = 24;
   // The app should receive the Windows license embedded in ACPI tables
   bool enable_oem_win_license_key = 25;
+  // Disable vTPM for the VM, this defaults to false according to the
+  // proto3 specification.
+  bool disable_vtpm = 26;
```

Comment:
I think we are trying to keep such bool fields with "enable_" semantics. The "disable_" one makes the code a little bit confusing.

Reply:
Ok, got it! I like the approach @milan-zededa suggested, but I'm also fine with the current approach.